ONSdigital/census-rh-ui
# Cr 1606 number in household

# Motivation and Context
Additional validation on 'Number in Household'

# What has changed
Text change in error message
Add upper limit to 'number in household'
Ensure that value of '0' doesn't cause an error (was causing an error for large-print)
Adds additional unit tests to cover new variations

No Cucumber updates required

# How to test?
Check on household paper-questionnaire fulfilment that number over 30 gives an error
Check on household paper-questionnaire fulfilment that entering '0' in household works when selecting 'large print'

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-1606
